core/src/main/java/META-INF/MANIFEST.MF

```diff
@@ -363,7 +363,7 @@ Require-Bundle: org.apache.commons.commons-codec;bundle-version=1.15.0,
  xmpcore;bundle-version=5.1.2.0002L,
  org.lucee.argon2;bundle-version=2.7.0,
  com.sun.jna;bundle-version=5.10.0
-Require-Extension: 7E673D15-D87C-41A6-8B5F1956528C605F;name=MySQL;label=MySQL;version=8.0.19,
+Require-Extension: 7E673D15-D87C-41A6-8B5F1956528C605F;name=MySQL;label=MySQL;version=8.0.28,
```

Comment:
can you please change this line to 
Require-Extension: 7E673D15-D87C-41A6-8B5F1956528C605F;name=MySQL;label=MySQL;version=8.0.30,